src/main/java/nextstep/qna/domain/Question.java

```diff
@@ -81,12 +88,41 @@ public boolean isDeleted() {
         return deleted;
     }
 
-    public List<Answer> getAnswers() {
+    public Answers getAnswers() {
         return answers;
     }
 
     @Override
     public String toString() {
         return "Question [id=" + getId() + ", title=" + title + ", contents=" + contents + ", writer=" + writer + "]";
     }
+
+    public DeleteHistories delete(NsUser loginUser) throws CannotDeleteException {
+        validateDelete(loginUser);
+        this.deleted = true;
+        deleteAnswers(loginUser);
+        return makeDeletehistories();
```

Comment:
아래와 같이 작성한다면 서비스에서 delete 메서드 하나만 호출해도 되겠네요
```suggestion
        validateDelete(loginUser);
        this.deleted = true;
        답변_삭제_히스토리 = answers.delete(loginUser);
        return makeDeletehistories(답변_삭제_히스토리);
```